lib/modules/leaks.js

```diff
@@ -13,6 +13,8 @@ const internals = {
         'clearInterval',
         'clearImmediate',
         'console',
+        'AbortController',
+        'AbortSignal',
```

Comment:
```suggestion
        'AbortSignal',
        'AggregateError',
```

`AggregateError` came with the V8 8.5/8.6 update: https://github.com/nodejs/node/pull/35415#issuecomment-700785147

The list from the most recent CITGM run with Node.js v15.0.0-proposal shows this failure for `@hapi/shot`:
```
The following leaks were detected:AbortController, AbortSignal, EventTarget, Event, MessageChannel, MessagePort, MessageEvent, AggregateError
```
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/2523/nodes=osx1014/testReport/junit/(root)/citgm/_hapi_shot_v5_0_4/